spec/models/sms/adapters/twilio_sms_adapter_spec.rb

```diff
@@ -54,6 +54,43 @@
     expect(msg.to).to be_nil
   end
 
+  context "broadcast message" do
+    let(:msg) { create(:sms_broadcast) }
+    let(:messages) { double(:twilio_messages, create: true) }
+    let(:client) { double(:twilio_client, messages: messages) }
+
+    before do
+      allow(msg).to receive(:recipient_numbers) { ["+12", "+34", "+56", "+78"] }
+      allow(@adapter).to receive(:client) { client }
```

Comment:
Please use `and_return(client)` since it's a bit clearer. Only use block if you need to compute something different each time.